package.json

```diff
@@ -11,43 +11,39 @@
     "yarn": ">=0.20.3"
   },
   "scripts": {
-    "start": "gulp serve",
-    "serve": "node dist/index.js",
-    "migrate": "./node_modules/.bin/sequelize db:migrate",
-    "migrate:undo": "./node_modules/.bin/sequelize db:migrate:undo:all",
-    "create_db": "cross-env NODE_ENV=test node dist/create_db.js",
-    "delete_db": "cross-env NODE_ENV=test node dist/delete_db.js",
-    "start:debug": "cross-env DEBUG=amida-auth-microservice:* yarn start",
-    "build": "gulp",
-    "lint": "esw -c .eslintrc *.js server config . --color",
-    "lint:watch": "yarn lint --watch",
+    "build": "babel src --out-dir dist",
+    "create_db": "sequelize db:create",
+    "delete_db": "sequelize db:drop",
+    "clean": "rimraf coverage dist",
+    "jenkins": "cross-env NODE_ENV=test yarn delete_db; cross-env NODE_ENV=test yarn create_db && cross-env NODE_ENV=test yarn migrate && cross-env NODE_ENV=test yarn test",
+    "lint": "esw src test --color",
     "lint:fix": "yarn lint --fix",
-    "test": "gulp && yarn create_db && cross-env NODE_ENV=test yarn migrate && cross-env NODE_ENV=test ./node_modules/.bin/mocha --ui bdd --reporter spec --colors --compilers js:babel-core/register server/tests --recursive -t 10000 && yarn delete_db",
-    "jenkins": "gulp && yarn delete_db && yarn create_db && cross-env NODE_ENV=test yarn migrate && cross-env NODE_ENV=test ./node_modules/.bin/mocha --ui bdd --reporter spec --colors --compilers js:babel-core/register server/tests --recursive -t 10000",
-    "test:watch": "yarn test --watch",
-    "test:coverage": "cross-env NODE_ENV=test ./node_modules/.bin/istanbul cover _mocha -- --ui bdd --reporter spec --colors --compilers js:babel-core/register server/tests --recursive",
-    "test:check-coverage": "yarn test:coverage && istanbul check-coverage",
+    "lint:watch": "yarn lint --watch",
+    "migrate": "sequelize db:migrate",
+    "migrate:undo": "sequelize db:migrate:undo:all",
+    "prepublish": "snyk protect",
     "report-coverage": "coveralls < ./coverage/lcov.info",
-    "seed": "gulp && cross-env DEBUG=amida-auth-microservice:* node dist/config/seed.js && gulp clean",
-    "prepublish": "snyk protect"
+    "seed": "sequelize db:seed:all",
+    "serve": "node dist/index.js",
+    "start": "nodemon --exec babel-node -- src/index.js",
+    "start:debug": "cross-env DEBUG=amida-auth-microservice:* yarn start",
+    "test": "cross-env NODE_ENV=test mocha --opts .mocharc",
```

Comment:
This is gonna create issues since it no longer creates a test db and then wiping it out.

Reply:
The intention was to separates `yarn jenkins` and `yarn test`. The jenkins command creates dbs, deletes dbs, runs the test etc.. The test command does exactly what's written on the tin -- run the tests.